package.json

```diff
@@ -25,27 +25,27 @@
   },
   "homepage": "https://github.com/seek-oss/eslint-config-seek#readme",
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.11.0",
-    "@typescript-eslint/parser": "^2.11.0",
+    "@typescript-eslint/eslint-plugin": "^3.0.0",
+    "@typescript-eslint/parser": "^3.0.0",
     "babel-eslint": "^10.0.3",
-    "eslint-config-prettier": "^6.7.0",
-    "eslint-import-resolver-node": "^0.3.2",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-import-resolver-node": "^0.3.3",
     "eslint-plugin-css-modules": "^2.11.0",
-    "eslint-plugin-cypress": "^2.7.0",
-    "eslint-plugin-flowtype": "^4.5.2",
-    "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-jest": "^23.1.1",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-react-hooks": "^2.3.0",
+    "eslint-plugin-cypress": "^2.10.3",
+    "eslint-plugin-flowtype": "^5.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jest": "^23.13.1",
+    "eslint-plugin-react": "^7.20.0",
+    "eslint-plugin-react-hooks": "^4.0.2",
     "find-root": "^1.1.0",
-    "typescript": "^3.7.3"
+    "typescript": "^3.9.3"
```

Comment:
Is this right? Should it be a `peerDependency`?

Reply:
It's probably not right. I'll make it an optional `peerDependency` and see what happens.